### ISSUE TITLE
Urls incompatibilities with django 1.10

### ISSUE THREAD

**Issue by SalahAdDin** [django-oscar/django-oscar-paypal#143]
Hi guys, i want report this:

``` bash
/home/salahaddin/Proyectos/demo-oscar/lib/python3.5/site-packages/paypal/express/urls.py:23: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  name='paypal-direct-payment'),

/home/salahaddin/Proyectos/demo-oscar/lib/python3.5/site-packages/paypal/express/dashboard/app.py:19: RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  name='paypal-express-detail'),
```

**Comment by sterago** [comment]
We are also running into this issue. Any update as to whether/when django-oscar-paypal will support django 1.10?

**Comment by SalahAdDin** [comment]
@sterago I don't know about it.

**Comment by sterago** [comment]
@codeinthehole can we expect django 1.10 support anytime soon and, in general, maintenance updates or should we look somewhere else as far as paypal integration with django-oscar is concerned?

**Comment by SalahAdDin** [comment]
@sasha0 Hey man.

**Comment by solarissmoke** [comment]
I've addressed this and other issues in  #159.
